lib/trino-parquet/src/test/java/io/trino/parquet/reader/flat/TestFlatColumnReader.java

```diff
@@ -77,12 +76,7 @@
     private static final PrimitiveField NULLABLE_FIELD = new PrimitiveField(INTEGER, false, new ColumnDescriptor(new String[] {}, TYPE, 0, 0), 0);
     private static final PrimitiveField FIELD = new PrimitiveField(INTEGER, true, new ColumnDescriptor(new String[] {}, TYPE, 0, 0), 0);
 
-    private final String[] fieldPath;
-
-    public TestFlatColumnReader(String[] fieldPath)
```

Comment:
"Enable Parquet tests" sounds gross

what about "Fix TestFlatColumnReaders test so it actually runs" ?

Reply:
+1

Reply:
yep, much better name